index.bs

```diff
@@ -67,8 +67,8 @@ enum StorageBucketDurability {
 };
 
 dictionary StorageBucketOptions {
-  boolean? persisted = null;
-  StorageBucketDurability? durability = null;
+  boolean persisted = false;
+  StorageBucketDurability durability = null;
```

Comment:
Presumably this second line was not meant to be changed here?

Reply:
Oops, thats correct. But it's ok, we removed the durability policy from the spec for now https://github.com/WICG/storage-buckets/pull/111